src/videojs-contrib-hls.js

```diff
@@ -273,6 +273,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper({tech});
```

Comment:
It is the only place in the project where we have a guaranteed reference to player, and if I moved it to the src function, I wouldn't have access to the tech.